spreadsheet/update/update_query.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+//
+// Use of this source code is governed by the terms of the Affero GNU General
+// Public License version 3.0 as published by the Free Software Foundation and
+// appearing in the file LICENSE included in the packaging of this file. A
+// commercial license can be purchased on https://unidoc.io.
+
+// package update contains definitions needed for updating references after removing rows/columns.
+
+package update
+
+// Update types constants.
+const (
+	REMOVE_COLUMN byte = iota
```

Comment:
Let's define a type for this
type UpdateAction type
and const (
  UpdateActionRemoveColumn UpdateAction = iota
)
that is more clear than using a byte type.